internal/swaggerDocs/SwaggerDocs.go

```diff
@@ -0,0 +1,77 @@
+// Package api QMS
```

Comment:
We normally have `snake_case` file names in Go, so this file should be renamed to `swagger_docs.go`.